@W-22438536 Auth commands (#1667)
salesforcecli/plugin-org: messages/org.auth.show-user-password.md

# summary

Show the stored password for an org's user.

# description

> Passwords are only available for orgs where a password was previously generated, such as by running "sf org generate password" or "sf org create user". Because passwords are sensitive credentials, this command prompts for confirmation before revealing it. Skip confirmation by specifying either the --no-prompt or --json flag.

Reviewer: I DM'd you about this, but I'll ask here too.  Is the idea that this CLI command can be used to show only the passwords that were generated in the DX project using a CLI command, such as `org generate password` or `org create user`?  In other words, if I create a user/password in Setup in the org, that password WON'T be showable with this command?  If so, how about this description:

---start---
This command shows only passwords that were generated locally in your DX project with either the `org generate password` or `org create user` CLI command.  If you generated a password for a user in Setup in your org, you can't show it with this command.  

Because passwords are sensitive credentials, this command prompts for confirmation before revealing it. Skip confirmation by specifying either the --no-prompt or --json flag.

---end---

Author: That's correct, if you created a user/pw in Setup and then authed to it locally, the pw would not be stored locally.

> Passwords are only available for orgs where a password was previously generated, such as by running "sf org generate password" or "sf org create user". Because passwords are sensitive credentials, this command prompts for confirmation before revealing it. Skip confirmation by specifying either the --no-prompt or --json flag.

Author: ```suggestion
This command shows only passwords that were generated locally in your DX project with either the "org generate password" or "org create user" CLI command. If you generated a password for a user in Setup in your org, you can't show it with this command.

Because passwords are sensitive credentials, this command prompts for confirmation before revealing it. Skip confirmation by specifying either the --no-prompt or --json flag.
```

# flags.no-prompt.summary

Skip the security warning and reveal the password without confirmation.

# prompt.show-user-password

You're about to reveal the password for "%s". Do you want to continue?

# warning.show-user-password

> This command exposes a user password. While a password alone is not sufficient to gain access to an org (additional factors like a security token or an enabled OAuth username-password flow are required), treat it as a sensitive credential and avoid sharing or logging it.

Reviewer: ```suggestion
This command exposes a user password. While a password alone isn't sufficient to gain access to an org (additional factors like a security token or an enabled OAuth username-password flow are required), treat it as a sensitive credential and avoid sharing or logging it.
```

# error.noPassword

> No password found for "%s". A password is only available if one was previously generated, such as by running "sf org generate password" or "sf org create user".

Reviewer: similar to previous comment.  If what I suspect is true, let's change this error to:

No password found for "%s". A password is available only if one was previously generated locally in your DX project by running either `sf org generate password` or `sf org create user`. If you generated a password in your org using Setup, it's not available to this command.

Author: ```suggestion
No password found for "%s". A password is available only if one was previously generated locally in your DX project by running either "sf org generate password" or "sf org create user". If you generated a password in your org using Setup, it's not available to this command.
```
